zipkin-web/src/main/resources/app/js/component_data/default.js

```diff
@@ -1,15 +1,41 @@
 import {component} from 'flight';
 import $ from 'jquery';
+import queryString from 'query-string';
+import chai from 'chai';
```

Comment:
maybe convert these two comments to code comments, as they are less likely to get lost.